docs/resources/sys_info.md.erb

```diff
@@ -50,3 +50,15 @@ For a full list of available matchers, please visit our [matchers page](https://
 The `hostname` matcher tests the host for which standard output is returned:
 
     its('hostname') { should eq 'value' }
+
+### manufacturer
+
+The `manufacturer` matcher tests the host for which standard output is returned:
+
+    its('manufacturer') { should eq 'value' }
+
+### model
+
+The `model` matcher tests the host for which standard output is returned:
+
+    its('model') { should eq 'value' }
```

Comment:
`its('model') { should eq 'SuperServer 2000' }`

Reply:
fixed